SuperTeamPicker/routes/cohorts.js

```diff
@@ -0,0 +1,109 @@
+const path = require('path');
+const multer = require('multer');
+const express = require('express');
+const router = express.Router();
+const knex = require('../db');
+
+const UPLOADS_DIR = 'uploads';
+const upload = multer({dest: path.join(__dirname, '..', 'public', UPLOADS_DIR)});
+
+// PATH: /cohorts/new VERB: GET Serves form for creating cohorts
+router.get('/new', (request, response) => {
+  // response.send('TEST POSTS NEW')
+  response.render('cohorts/new');
+});
+
+// localhost:7000/cohorts/:id
+router.get('/:id', (request, response) => {
+  const id = request.params.id;
+  const members = request.query.members;
+  const method = request.query.method;
+  const quantity = request.query.quantity;
+
+  function shuffleArray(array) {
+
+      for (var i = array.length - 1; i > 0; i--) {
+          var j = Math.floor(Math.random() * i);
```

Comment:
Going forward, you should try to only use let and const since you can have better scope control with variables than using var.